arches/app/etl_modules/import_single_csv.py

```diff
@@ -280,12 +280,22 @@ def populate_staging_table(self, loadid, graphid, has_headers, fieldnames, csv_f
                                 config = current_node.config
                                 config["path"] = temp_dir
                                 value = (
-                                    datatype_instance.transform_value_for_tile(source_value, **config) if source_value is not None else None
+                                    datatype_instance.transform_value_for_tile(source_value, **config)
+                                    if source_value is not None and source_value != ""
+                                    else None
                                 )
                                 errors = datatype_instance.validate(value, nodeid=node, path=temp_dir)
                             else:
-                                value = datatype_instance.transform_value_for_tile(source_value) if source_value is not None else None
-                                errors = datatype_instance.validate(value)
+                                try:
+                                    config = current_node.config
+                                except TypeError:
+                                    config = {}
```

Comment:
why the try/except here? is it possible to check for the current_node config a different way?

Reply:
Yes you are right. There should always be the config. So, remove try and move the line to above combine with file type. Thanks.